components/Search.js

```diff
@@ -29,9 +32,14 @@ export default function Search({ data, func, attrs, placeholder }) {
   };
 
   return (
-    <div>
-      <input value={inputValue} onChange={handleChange} placeholder={placeholder || 'Search'} />
-      <button onClick={handleClick}>Clear</button>
+    <>
+    <div class="d-flex justify-content-center" style={{alignItems: 'center', marginBottom: '20px'}}>
+      <input style={{border: '1px solid lightgray', borderRadius: '30px', width: '40vh', height: '35px', marginLeft: '10px', paddingLeft: '20px'}} value={inputValue} onChange={handleChange} placeholder={placeholder || 'Search'} />
+      <p onClick={handleClick} style={{margin: '10px', color: 'red'}}>Reset</p>
```

Comment:
Let's add a `style={{cursor: 'pointer'}}` to this p tag

Reply:
On it!